connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -205,6 +208,7 @@ private void syncTopicAcls()
         List<AclBinding> bindings = listTopicAclBindings().stream()
             .filter(x -> x.pattern().resourceType() == ResourceType.TOPIC)
             .filter(x -> x.pattern().patternType() == PatternType.LITERAL)
+            .filter(x -> !(x.entry().permissionType() == AclPermissionType.ALLOW && x.entry().operation() == AclOperation.WRITE))
```

Comment:
IMO, this is complex enough to warrant a named method, similar to shouldReplicateTopic below. In-line lambdas like this should be absolutely trivial -- a method call, a bit of arithmetic, a comparison... this expression is just complex enough that I had to read it a couple times, which tells me a method name would be better here.